### ISSUE TITLE
Android top bar should be white, not red

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#921]
Right now there's a big way where the app doesn't match designs: the top bar is red and not white. 

Having a white bar works better because it blends continuously with the background. 

We also might want to nudge the top bar content upwards if the resulting white space is larger than imagined in the designs, which I *think* it will be.

**Comment by holmesworcester** [comment]
This is done, it seems.

**Comment by kingalg** [comment]
Yes, it's definitely white in both android and ios. I'm moving this to "done".
